build_tools/circle/build_test_arm.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -e
+set -x
+
+UNAMESTR=`uname`
+
+setup_ccache() {
+    echo "Setting up ccache"
+    mkdir /tmp/ccache/
+    which ccache
+    for name in gcc g++ cc c++ x86_64-linux-gnu-gcc x86_64-linux-gnu-c++; do
+      ln -s $(which ccache) "/tmp/ccache/${name}"
+    done
+    export PATH="/tmp/ccache/:${PATH}"
+    ccache -M 256M
+}
+
+# imports get_dep
+source build_tools/shared.sh
+
+sudo add-apt-repository --remove ppa:ubuntu-toolchain-r/test
+sudo apt-get update
+sudo apt-get install python3-scipy python3-matplotlib libopenblas-base \
+                     python3-virtualenv ccache
+python3 -m virtualenv --system-site-packages --python=python3 testenv
+source testenv/bin/activate
```

Comment:
Just in case pip fix bugs in the future, let's use the latest version in the venv:

```suggestion
source testenv/bin/activate
pip install --upgrade pip
```